backend/capellacollab/sessions/injection.py

```diff
@@ -3,42 +3,60 @@
 
 import logging
 
+import pydantic
 import requests
 
 from capellacollab import core
 from capellacollab.config import config
+from capellacollab.core import pydantic as core_pydantic
 
 log = logging.getLogger(__name__)
 
 
-def get_last_seen(sid: str) -> str:
-    """Return project session last seen activity"""
+# Has to be in here because of circular imports
+class IdleState(core_pydantic.BaseModel):
+    available: bool
+    idle_for_minutes: float | None = pydantic.Field(default=None)
```

Comment:
```suggestion
    idle_for_minutes: int | None = pydantic.Field(default=None)
```

We collect the idletime only every 30 seconds, therefore a float would be inaccurate.